spec/lib/pantry/helpers/category_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Pantry::Helpers::CategoryHelper, type: :helper do
```

Comment:
I'd avoid assigning the `type: :helper` here, as that has special meaning in Rails apps.  You could assign your own, arbitrary type to something like `type: :categorization` if you ever needed to inject behavior into all the tests around your category helper code.  Otherwise, you can just drop the type.